package.json

```diff
@@ -58,7 +58,7 @@
     "redux-actions": "^2.6.5"
   },
   "optionalDependencies": {
-    "immer": ">= 1.0.0",
+    "immer": "^1.0.0",
```

Comment:
Current immer version is 9.0, I'd like to consider not locking major version since the function that typed-actions uses is still there with the same signature and still exported using the same export method. This can help avoid npm warning during `npm install`

Reply:
reverted, and locked `immer` dev version